config/Run3_2022EE/samples.yaml

```diff
@@ -44,3 +38,990 @@ JetMET_Run2022F:
+ZH_Hbb_Zll_ext1:
+  sampleType: VH
+  crossSection: ZH_Hbb_Zll
+  generator: powheg
+  miniAOD: /ZH_Hto2B_Zto2L_M-125_TuneCP5_13p6TeV_powheg-pythia8/Run3Summer22EEMiniAODv4-130X_mcRun3_2022_realistic_postEE_v6_ext1-v2/MINIAODSIM
+ZH_Hbb_Zqq:
+  sampleType: VH
+  crossSection: ZH_Hbb_Zqq
+  generator: powheg
+  miniAOD: /ZH_Hto2B_Zto2Q_M-125_TuneCP5_13p6TeV_powheg-pythia8/Run3Summer22EEMiniAODv4-130X_mcRun3_2022_realistic_postEE_v6-v2/MINIAODSIM
+ZH_Hbb_Zqq_ext1:
+  sampleType: VH
+  crossSection: ZH_Hbb_Zqq
+  generator: powheg
+  miniAOD: /ZH_Hto2B_Zto2Q_M-125_TuneCP5_13p6TeV_powheg-pythia8/Run3Summer22EEMiniAODv4-130X_mcRun3_2022_realistic_postEE_v6_ext1-v3/MINIAODSIM
+
+############## W ##############
+WtoLNu_0J_amcatnloFXFX:
+  sampleType: W
+  crossSection: WtoLNu_0J_amcatnloFXFX
+  generator: amcatnloFXFX
+  miniAOD: /WtoLNu-2Jets_0J_TuneCP5_13p6TeV_amcatnloFXFX-pythia8/Run3Summer22EEMiniAODv4-130X_mcRun3_2022_realistic_postEE_v6-v3/MINIAODSIM
+WtoLNu_1J_amcatnloFXFX:
+  sampleType: W
+  crossSection: WtoLNu_1J_amcatnloFXFX
+  generator: amcatnloFXFX
+  miniAOD: /WtoLNu-2Jets_1J_TuneCP5_13p6TeV_amcatnloFXFX-pythia8/Run3Summer22EEMiniAODv4-130X_mcRun3_2022_realistic_postEE_v6-v2/MINIAODSIM
+WtoLNu_1J_madgraphMLM:
+  sampleType: W
+  crossSection: WtoLNu_1J_madgraphMLM
+  generator: madgraphMLM
+  miniAOD: /WtoLNu-4Jets_1J_TuneCP5_13p6TeV_madgraphMLM-pythia8/Run3Summer22EEMiniAODv4-130X_mcRun3_2022_realistic_postEE_v6-v2/MINIAODSIM
+WtoLNu_2J_amcatnloFXFX:
+  sampleType: W
+  crossSection: WtoLNu_2J_amcatnloFXFX
+  generator: amcatnloFXFX
+  miniAOD: /WtoLNu-2Jets_2J_TuneCP5_13p6TeV_amcatnloFXFX-pythia8/Run3Summer22EEMiniAODv4-130X_mcRun3_2022_realistic_postEE_v6-v2/MINIAODSIM
+WtoLNu_2J_madgraphMLM:
+  sampleType: W
+  crossSection: WtoLNu_2J_madgraphMLM
+  generator: madgraphMLM
+  miniAOD: /WtoLNu-4Jets_2J_TuneCP5_13p6TeV_madgraphMLM-pythia8/Run3Summer22EEMiniAODv4-130X_mcRun3_2022_realistic_postEE_v6-v2/MINIAODSIM
+WtoLNu_3J_madgraphMLM:
+  sampleType: W
+  crossSection: WtoLNu_3J_madgraphMLM
+  generator: madgraphMLM
+  miniAOD: /WtoLNu-4Jets_3J_TuneCP5_13p6TeV_madgraphMLM-pythia8/Run3Summer22EEMiniAODv4-130X_mcRun3_2022_realistic_postEE_v6-v2/MINIAODSIM
+WtoLNu_4J_madgraphMLM:
+  sampleType: W
+  crossSection: WtoLNu_4J_madgraphMLM
+  generator: madgraphMLM
+  miniAOD: /WtoLNu-4Jets_4J_TuneCP5_13p6TeV_madgraphMLM-pythia8/Run3Summer22EEMiniAODv4-130X_mcRun3_2022_realistic_postEE_v6-v2/MINIAODSIM
+WtoLNu_amcatnloFXFX:
+  sampleType: W
+  crossSection: WtoLNu
+  generator: amcatnloFXFX
+  miniAOD: /WtoLNu-2Jets_TuneCP5_13p6TeV_amcatnloFXFX-pythia8/Run3Summer22EEMiniAODv4-130X_mcRun3_2022_realistic_postEE_v6-v2/MINIAODSIM
+WtoLNu_madgraphMLM:
+  sampleType: W
+  crossSection: WtoLNu
+  generator: madgraphMLM
+  miniAOD: /WtoLNu-4Jets_TuneCP5_13p6TeV_madgraphMLM-pythia8/Run3Summer22EEMiniAODv4-130X_mcRun3_2022_realistic_postEE_v6-v2/MINIAODSIM
+WtoLNu_madgraphMLM_ext1:
+  sampleType: W
+  crossSection: WtoLNu_madgraphMLM
```

Comment:
`WtoLNu_madgraphMLM` cross-section is not defined

Reply:
Error with the _ext files, fixed as mentioned before